README.md

```diff
@@ -1,3 +1,212 @@
 [![Build Status](https://travis-ci.org/saagie/whyat-js.svg?branch=master)](https://travis-ci.org/saagie/whyat-js)
 # whyat-js
-Y@ (whyat) helps you to track your user’s action within your application and store it directly in the data source of your choice
+Y@ (whyat) helps you to track your user’s actions within your application and store it directly in the data source of your choice
+
+Use with [Y@ Server](https://github.com/saagie/whyat-server) to store your user's events.
+
+## Installation 
+```$ npm install whyat-js --save```
```

Comment:
explain that the user should also install peer dependencies, and call `require('babel-polyfill`)` in their application.
saying that makes me feel not well... :(